api/v1/controllers/RegistrationController.js

```diff
@@ -377,6 +396,9 @@ router.put('/attendee/decision/:id',  middleware.request(requests.AttendeeDecisi
 router.put('/attendee/:id(\\d+)', middleware.request(requests.AttendeeRequest),
 	middleware.permission(roles.ORGANIZERS), updateAttendeeById);
 
+router.put('/sendlist', middleware.request(requests.SendListRequest), 
```

Comment:
Is this route only for acceptance waves? I was under the impression that we want to use this for any list